migration.sql

```diff
@@ -0,0 +1,61 @@
+USE tomo_db;
```

Comment:
Excellent work! Just one observation. There's no need to create a migration, since everything is created by Spring when you annotate your models.